recipe/meta.yaml

```diff
@@ -106,7 +106,7 @@ requirements:
     - libwebp
     - qt-main                        # [not osx and not ppc64le]
     - zlib
-    - openvino                       # [not ppc64le]
+    - libopenvino-dev                # [not ppc64le]
```

Comment:
`openvino` has added Python API since 2023.1 release, which is not required for OpenCV.
`openvinolib-dev` is a subpackage, responsible for C++ part, which is enough here.

Reply:
thanks.